crates/cdk/src/nuts/nut06.rs

```diff
@@ -59,9 +59,9 @@ pub struct MintInfo {
     /// long description
     #[serde(skip_serializing_if = "Option::is_none")]
     pub description_long: Option<String>,
-    /// contact methods to reach the mint operator
+    /// Contact info
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub contact: Option<Vec<Vec<String>>>,
+    pub contact: Option<Vec<ContactInfo>>,
```

Comment:
Add custom deserialize function that converts old format from mints to new during transition